katsdpcontroller/scheduler.py

```diff
@@ -3296,9 +3300,13 @@ async def run(node_set):
                                 desc = node.name
                             else:
                                 desc = node.name + ' (and {} others)'.format(len(node_set) - 1)
-                            logger.info('Skipping %s because %s failed', desc, dep.name)
-                            self.task_stats.batch_tasks_skipped(len(node_set))
-                            raise TaskSkipped(node) from None
+                            if critical:
+                                logger.info('Skipping %s because %s failed', desc, dep.name)
+                                self.task_stats.batch_tasks_skipped(len(node_set))
+                                raise TaskSkipped(node) from None
+                            else:
+                                logger.debug('Continuing with %s after non-critical %s failed',
```

Comment:
Torn a bit here - I feel that this should actually be promoted all the way to WARNING. The logic is that I would always like a WARNING level event when something occurs that could lead to output data or products not being what you would expect. In this case for instance the report could be missing channels. 

I realise that the failure event leading here in the first place will log useful things... And this is probably overthinking a simple log message...

Reply:
> I realise that the failure event leading here in the first place will log useful things

That was my thinking - having a higher log level here is just going to log everything twice, and the first failure will have more useful information. I also aim to have the report itself indicate channels for which no statistics arrived (for channels where everything is flagged I'll still have the pipeline store something to indicate that, so that it can be distinguished from failures).

Reply:
Fair enough